src/test/java/com/wikia/webdriver/testcases/specialpagestests/CuratedContentTests.java

```diff
@@ -53,4 +53,23 @@ public void curatedContent002_saveWithImage() {
     cc.clickSave();
     cc.verifySuccesfulSave();
   }
+
+  @CreationTicket(ticketID = "CONCF-1073")
+  @Test(groups = {"CuratedContent003", "CuratedContent"})
+  public void curatedContent003_open() {
+    WikiBasePageObject base = new WikiBasePageObject(driver);
```

Comment:
this is not necessary, open method should be in `SpecialCuratedContentPageObject`